cmd/hdfs/test/put.bats

```diff
@@ -64,6 +64,33 @@ mkdir /_test_cmd/put/existing.txt: file already exists
 OUT
 }
 
+@test "put from stdin" {
+  run bash -c "cat $ROOT_TEST_DIR/test/mobydick.txt | $HDFS put - /_test_cmd/put_stdin/mobydick_stdin.txt"
+  assert_success
+
+  run bash -c "$HDFS cat /_test_cmd/put_stdin/mobydick_stdin.txt > $BATS_TMPDIR/mobydick_stdin_test.txt"
+  assert_success
+
+  SHA=`shasum < $ROOT_TEST_DIR/test/mobydick.txt | awk '{ print $1 }'`
+  assert_equal $SHA `shasum < $BATS_TMPDIR/mobydick_stdin_test.txt | awk '{ print $1 }'`
+}
+
+@test "put from stdin into existing file" {
```

Comment:
How does the normal `hadoop -fs` behave in this case, out of curiosity?

Reply:
Just had a try: `put: `/tmp/testme': File exists`